### ISSUE TITLE
[Improvement] Show notification when a user starts a secret chat

### ISSUE THREAD

**Issue by diazbastian** [telegramdesktop/tdesktop#1871]
### Steps to reproduce
1. Log in on Telegram Desktop
2. Ask a friend to start a secret chat from a mobile device
3. Wait

### Actual behaviour
Nothing happens. If you do not have to hand the smartphone with internet and feel the notice, we do not realize that a friend wants to start a secret chat. Nothing happens. We did not realize that a friend started a secret chat, but to pay attention to the smartphone/tablet.

### Expected behaviour
A notification shows that a user has started a secret chat with us. This helps to pay attention and use a compatible device to continue communication.

### Configuration
Fedora 23 64 bit (gnome 3.18)

**Version of Telegram Desktop:**
0.9.40 stable

###Additional Information
Related to #871

**Comment by ppparadox** [comment]
+1, a good balance between security and usability. Not the message but at least the notification so you can pick up the "secure" device.

**Comment by haevalencia** [comment]
Losing a conversation because it is not notified is critical. If you will not have secret chats on Desktop in the short term this seems the best solution.

If it's not something native to the app it could be the same Telegram notifications service or an official bot.

**Comment by kyanha** [comment]
Why won't there be secret chats on Desktop?

**Comment by erotavlas85** [comment]
What about adding secret chats on desktop? In the case of secret chats the synchronization is not necessary since every device can remain independent from the others.

**Comment by Aokromes** [comment]
still wanted.

**Comment by alfredonodo** [comment]
Yes, this is one of the most important feature that is missing on telegram desktop together with instant view.
